### PR TITLE
[f5cloudexporter] Updated for upcoming client auth extension changes

### PR DESCRIPTION
**Pull request by gramidt**
**Description:** 
Updated f5cloudexporter for upcoming client auth extension changes

**Link to tracking Issue:**
#3507 

**Testing:**
Unit tests

**Documentation:**
Unit tests

### CODE REVIEW

File: exporter/f5cloudexporter/config.go [hunk 1]
```diff
@@ -31,7 +31,7 @@ type Config struct {
 	Source string `mapstructure:"source"`
 
 	// AuthConfig represents the F5 Cloud authentication configuration options.
-	AuthConfig AuthConfig `mapstructure:"auth"`
+	AuthConfig AuthConfig `mapstructure:"f5cloud_auth"`
```
Comment (bogdandrutu) [inline exporter/f5cloudexporter/config.go:34]: This is a breaking change, why is this needed? Can this be avoided?

Comment (gramidt) [reply exporter/f5cloudexporter/config.go:34]: That is correct, but it can't be avoided with the implementation that was chosen for Client Auth Extension, since it squashes into a he configured helpers. I have already communicated to users of it and internal F5 teams that this will break it and that we will be deprecating this in favor of an F5 Cloud Auth Extension + gRPC OTLP.

Comment (bogdandrutu) [reply exporter/f5cloudexporter/config.go:34]: Add an entry to the changelog then to explain it and motivation of it.

**Review by bogdandrutu — APPROVED** [review]
